### DISCUSSION TITLE
Can I get the starstruck badge even if my repository is private?

### DISCUSSION THREAD

**Discussion by janzengo** [community/community#48960] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Howdy! I hope y'all are doing great. My question's pretty straightforward. I had been searching for the same question across the platform but got no luck of finding any. So, is it possible for me to obtain the Starstruck Badge, having 16 stars on a private repository?

**Comment by Anmol-Baranwal** [comment, marked as answer]
It will not be visible for anyone to visit.
And how can someone star a repository if it is private, which can only be done by collaborators. So, what is the overall purpose?
It doesn't make sense right.

> Stars are lost when converting a public repo to private one.

**Reply by janzengo** [reply]
Oh I see, that was rather a stupid question lol. I appreciate the response tho!

**Reply by Anmol-Baranwal** [reply]
It is more important to be curious, even if it seems stupid to you.
